cypress/integration/Menu.spec.ts

```diff
@@ -119,6 +119,24 @@ describe('Menu', () => {
         });
       });
 
+      context('when the Fourth item is tabbed', () => {
+        beforeEach(() => {
+          cy.findByRole('menuitem', {name: 'Fourth Item'}).tab();
+        });
```

Comment:
Why does the fourth item need focus? Hitting tab will close no matter which option is currently selected.

```suggestion
      context('when the tab key is pressed', () => {
        beforeEach(() => {
          cy.focused().tab();
        });
```